packages/mutation-testing-report-schema/src/mutation-testing-report-schema.json

```diff
@@ -73,6 +103,18 @@
                   "description": "Unique id, can be used to correlate this mutant with other reports.",
                   "examples": ["321321"]
                 },
+                "description": {
+                  "type": "string",
+                  "description": "Description of the applied mutation.",
+                  "examples": ["removed call to java/io/Writer::write"]
+                },
+                "killedBy": {
```

Comment:
What if we cannot determine the exact test, or if there are multiple that kill this mutant?

Reply:
It's optional, so if you cannot determine you simply leave it out. If there are multiple tests, I would suggest using the first one here? I think it is a good practice to use "bail" mode for mutation testing, so stop testing as soon as one test fails.

Reply:
Why not just list all tests that killed a mutant? Bail mode cannot be used in all mutation testing modes for stryker.net currently.

Reply:
It's mostly because I want to add a view later listing all tests that didn't kill a mutant. I want that list to not have false-negatives. So in my opinion every test should be able to kill unique mutants if that makes sense. If we would allow a list here, this new view might work differently depending on the way "killed-by" was reported.

I'm not strongly against it, just one man's opinion.

Reply:
We had a conversation about it and let's make it an array.

It's up to the mutation testing framework to decide to use "bail" mode or not, but that should still be a best practice IMO.

We've also discussed merging this array into the `testCoverage` array but decided to keep them separate. A mutation testing framework might not implement test coverage, but still want to report the `killedBy`.